sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/aio/_azure_appconfiguration_client_async.py

```diff
@@ -4,17 +4,18 @@
 # license information.
 # -------------------------------------------------------------------------
 import binascii
-from typing import Any, Optional, Mapping, Union
+from typing import Any, Dict, List, Mapping, Optional, Union
 from requests.structures import CaseInsensitiveDict
```

Comment:
is there a particular reason to use CaseInsensitiveDict from requests ? There is one in azure core itself that is similar.

Reply:
Updated to use the one from azure-core.